tests/test_updater_delegation_graphs.py

```diff
@@ -362,6 +362,96 @@ def test_safely_encoded_rolenames(self) -> None:
         exp_calls = [(quoted[:-5], 1) for quoted in roles_to_filenames.values()]
         self.assertListEqual(self.sim.fetch_tracker.metadata, exp_calls)
 
+    hash_bins_graph: utils.DataSet = {
+        "delegations": DelegationsTestCase(
+            delegations=[
+                TestDelegation(
+                    "targets",
+                    "b8-bf",
+                    paths=None,
+                    path_hash_prefixes=[
+                        "b8",
+                        "b9",
+                        "ba",
+                        "bb",
+                        "bc",
+                        "bd",
+                        "be",
+                        "bf",
+                    ],
```

Comment:
I suppose one way to make this shorter is to use e.g.
```suggestion
                    path_hash_prefixes=["8", "9", "a", "b"],
```
This way you could have four delegations (0-3, 4-7, 8-b, c-f) that actually cover all possible path hashes

Reply:
+1 for this. Those linter requirements are sometimes really hard for reading.